lua/build/root.build

```diff
@@ -0,0 +1,15 @@
+c.std = 99
+
+using c
+
+h{*}: extension = h
+c{*}: extension = c
+
+test.target = $c.target
+
+# Configuration Variables
+
+# Decides whether to use GNU `readline` library
+# for the Lua interpreter `exe{lua}`.
+#
```

Comment:
nit: reflow comment to textwidth (with a maximum width of 80); Perhaps shorten text to avoid awkward line breaks?

```suggestion
# Decides whether to use GNU `readline` library for the Lua interpreter
# `exe{lua}`.
#
```
```suggestion
# Whether to use GNU `readline` library for the Lua interpreter `exe{lua}`.
#
```